src/git/commit_filter_test.go

```diff
@@ -185,10 +185,20 @@ func TestCommitFilter_NewCommitFilter(t *testing.T) {
 			},
 			expectedError: git.ErrDirNotValid,
 		},
+		{
+			name: "Directory_for_files_Throw_No_Error",
```

Comment:
🤔 but it is throwing an error, isn't it?

Reply:
copy/paste 😮‍💨